ee/modules/600-flant-integration/monitoring/prometheus-rules/sending-alerts.yaml

```diff
@@ -7,7 +7,7 @@
           max by (pod_ip) (
               label_replace(
                 max by (pod, alertmanager) (
-                  (rate(prometheus_notifications_errors_total{pod="prometheus-main-0"}[5m]) / rate(prometheus_notifications_sent_total{pod="prometheus-main-0"}[5m])) * 100) > 0,
+                  (rate(prometheus_notifications_errors_total{pod=~"prometheus-main-.*"}[5m]) / rate(prometheus_notifications_sent_total{pod=~"prometheus-main-.*"}[5m])) * 100) > 0,
```

Comment:
More accurate
```
rate(prometheus_notifications_errors_total[5m]) 
/ on (pod, alertmanager) (rate(prometheus_notifications_sent_total[5m])
```
instead of `max by (pod, alertmanager) ()`.

You do not need a pod label selector here.